tools/travis/run_autopkgtests.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+#
+# Copyright (C) 2017 Canonical Ltd
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+# Run snapcraft autopkgtests.
+# Arguments:
+#   suite: The test suite to run.
+
+set -ev
+
+if [ "$#" -ne 2 ] ; then
+    echo "Usage: "$0" <distro> <test>"
+    exit 1
+fi
+
+distro="$1"
+test="$2"
+
+lxc="/snap/bin/lxc"
+
+script_path="$(dirname "$0")"
+project_path="$(readlink -f "$script_path/../..")"
+
+"$script_path/setup_lxd.sh"
+"$script_path/run_lxd_container.sh" test-runner
+$lxc file push --recursive $project_path test-runner/root/
+# Workaround for
+# - Setup snap "core" (2462) security profiles (cannot reload udev rules: exit status 2
```

Comment:
this comment is out of context, should explain its use twice below

Reply:
On #1643 I've duplicated the comment and moved it down. Is that what you meant?

Reply:
yes